### DISCUSSION TITLE
cannot import name 'Accuracy'

### DISCUSSION THREAD

**Discussion by Sarmadfismael** [Lightning-AI/torchmetrics#470] [Q&A] [answered]
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-50-89059b71aef0> in <module>
----> 1 from torchmetrics.classification import Accuracy
      2 
      3 train_accuracy = Accuracy()
      4 valid_accuracy = Accuracy(compute_on_step=False)

ImportError: cannot import name 'Accuracy' from 'torchmetrics.classification' (unknown location)

**Comment by phiyodr** [comment, marked as answer]
You need to import Accuracy like this (no need for `.classification`):
```python
from torchmetrics import Accuracy 
```

**Reply by SkafteNicki** [reply]
Actually, both import methods should work:
![Capture](https://user-images.githubusercontent.com/24896311/130189755-e03b0d9c-f11e-44bc-8d04-de6b93648fcb.PNG)
@Sarmadfismael not sure why you are getting such error as we have not changed the import pattern for a very long time.
